<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MuppetMainWindow</class>
 <widget class="QMainWindow" name="MuppetMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1182</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MuppetMainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../muppeticons.qrc">
    <normaloff>:/icons/dllicon.png</normaloff>:/icons/dllicon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1182</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionCascade"/>
    <addaction name="actionTile"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSaveSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindows"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="ControlToolBar">
   <property name="windowTitle">
    <string>MainControlBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="HostDockWidget">
   <property name="windowTitle">
    <string>Host Control</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="HostnameLineEdit">
         <property name="toolTip">
          <string>NHostname of RASPI with DABC command server</string>
         </property>
         <property name="text">
          <string>RA-9</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="PortSpinBox">
         <property name="toolTip">
          <string>TCP port of DABC  command server</string>
         </property>
         <property name="maximum">
          <number>99999</number>
         </property>
         <property name="value">
          <number>54321</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="TestButton">
         <property name="toolTip">
          <string>Test connection to DABC command server</string>
         </property>
         <property name="text">
          <string>Test</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="Connectlabel">
       <property name="text">
        <string>Not Connected</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>463</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="ModeDockWidget">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>ModeControl</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="ModesgroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Mode</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QCheckBox" name="checkBox_AA">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If checked values will be applyed directly when GUI elements are changed &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>AutoApply</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="HexBox">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Check for hexadecimal numbers mode&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Hexmode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="DebugBox">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Check for verbose debug mode of gosipcmd&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Verbose</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="LogDockWidget">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Output</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QPlainTextEdit" name="TextOutput">
       <property name="font">
        <font>
         <family>Andale Mono</family>
        </font>
       </property>
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="SFPToolBar">
   <property name="windowTitle">
    <string>SFP Controls</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea|Qt::RightToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="../muppeticons.qrc">
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../muppeticons.qrc">
     <normaloff>:/icons/filesave.png</normaloff>:/icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../muppeticons.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../muppeticons.qrc">
     <normaloff>:/icons/mbslogo.png</normaloff>:/icons/mbslogo.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="text">
    <string> Designer Cascade</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="text">
    <string>Tile</string>
   </property>
  </action>
  <action name="actionSaveSettings">
   <property name="text">
    <string>SaveSettings</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../muppeticons.qrc"/>
 </resources>
 <connections/>
</ui>
